hackathon_site/hackathon_site/tests.py

```diff
@@ -22,7 +22,11 @@ def setUp(self):
             email="foo@bar.com",
         )
 
-    def _login(self):
+    def _login(self, permissions=None):
+        if permissions is not None:
+            for permission in permissions:
```

Comment:
Does this work if you just do `self.user.permissions.add(permissions)`? `permissions` should be a queryset, so it should be able to add them all in a single query. Will help speed up our tests a tiny amount.

Reply:
I have tried this. for .add(), it only accepts 1 variable at a time. If it got more than 2 querysets (when I use Permission.objects.select(), it will just through out an error), I can attach the screenshot here later

Reply:
so I add a for loop there, and the error is solved

Reply:
Ah my bad, you're absolutely right!